### ISSUE TITLE
Last pre-release build

### ISSUE THREAD

**Issue by Lis105** [4lex4/scantailor-advanced#134]
Please compile the executable file with the latest version of the code for Windows 64.
Judging by the questions and answers, many errors have already been fixed from version 2019.8.16

**Comment by ElTigreCS** [comment]
Yes, please compile the exe file with the latest version of the code for Windows x64. Many thanks @4lex4 and others contributors for all you're doing on STA software. Amazing! :)

**Comment by FinnNk** [comment]
Yes, this would be great - I've built the 32bit version but the 64bit build is a bit fiddly for people (like me) not familiar with the C++ toolchain.

**Comment by darnn** [comment]
Somewhat belatedly:
https://github.com/4lex4/scantailor-advanced/issues/187
